projects/ngrx.io/content/guide/component/let.md

```diff
@@ -5,19 +5,21 @@ The `*ngrxLet` directive serves a convenient way of binding observables to a vie
 
 ## Usage
 
-The `*ngrxLet` directive is provided through the `LetModule`.
-To use it, add the `LetModule` to the `imports` of your standalone component or NgModule:
+The `*ngrxLet` directive is a standalone directive.
+To use it, add the `LetDirective` to the `imports` of your standalone component or NgModule. Alternatively, you can use `LetModule`, however, keep in mind that the `LetModule` is deprecated.
+
+Using the standalone `LetDirective`:
```

Comment:
```suggestion
To use it, add the `LetDirective` to the `imports` of your standalone component or NgModule:
```

Below the example, we can add the following section:

```
<div class="alert is-critical">

The `*ngrxLet` directive can be also used by importing the `LetModule`.
However, the `LetModule` is deprecated in favor of the standalone `LetDirective`.
See the [migration guide](guide/migration/v16#letmodule) for more information.

</div>
```

We also need to add a section about deprecated `LetModule` and `PushModule` to the `v16.md` file. Check the [v14 migration guide](https://ngrx.io/guide/migration/v14#reactivecomponentmodule) to see how it should be done.

Reply:
Added, also in the v16.md